docs/topics/braintree.rst

```diff
@@ -10,16 +10,47 @@ Calls braintree `ClientToken.generate <https://developers.braintreepayments.com/
 
 .. http:post:: /braintree/token/generate/
 
-    **Request**
+    :>json string token: the token returned by Braintree.
+    :status 200: token successfully generated.
+
+Create a customer
+-----------------
+
+Creates a customer in Braintree and the corresponding buyer in solitude. If the
+buyer exists in solitude already, it is not created. If the customer exists in
+Braintree already, it is not created.
 
-    No parameters.
+.. http:post:: /braintree/customer/
 
-    **Response**
+    :<json string uuid: the uuid of the buyer in solitude.
```

Comment:
![screenshot 2015-04-27 15 56 16](https://cloud.githubusercontent.com/assets/74699/7359677/12f866d8-ecf6-11e4-87b5-97d75df61e7b.png)
Sphinx prepends Request or Response if it see < or > , so it seemed redundant.